docs/markdown/Commands.md

```diff
@@ -149,6 +150,26 @@ optional arguments:
   --clean                               Clean the build directory.
   -C BUILDDIR                           The directory containing build files to
                                         be built.
+  --ninja-args NINJA_ARGS               Arguments to pass to `ninja` (applied
+                                        only on `ninja` backend).
+  --vs-args VS_ARGS                     Arguments to pass to `msbuild` (applied
+                                        only on `vs` backend).
+```
+
+`BACKEND-args` use the following syntax:
+
+If you only pass a single string, then it is considered to have all values separated by commas. Thus invoking the following command:
+
+```
+$ meson compile --ninja-args=foo,bar
```

Comment:
I would use a real example `--ninja-args=-k 0`, or is it `--ninja-args="-k 0"` or  `--ninja-args="['-k', '0']"`` ??

Reply:
Added a better(?) example.